docs(ee/vitals) add docker link, note for exporter count and remove API rules (#1488)
Kong/docs.konghq.com: app/enterprise/0.36-x/admin-api/vitals/vitals-prometheus-strategy.md

Then start StatsD exporter with

> ```bash

Reviewer: Will removing this option affect previous versions? Was it extraneous to begin with?

Author: Added back for backward compatibility 👍

StatsD exporter can run either on a separate node from Kong (to avoid resource
competition with Kong), or on the same host with Kong (to reduce unnecessary
network overhead). Note that feeding metrics from multiple Kong nodes into a single

> StatsD exporter is currently not supported, as this will result in metrics conflict.

Reviewer: ```suggestion
StatsD exporter is currently not supported, as it would result in a metrics conflict.
```
